services/data-info/src/data_info/routes/domain/common.clj

```diff
@@ -11,11 +11,19 @@
 (def DataIdPathParam (describe UUID "The data item's UUID"))
 
 (s/defschema Paths
-  {:paths (describe [NonBlankString] "A list of IRODS paths")})
+  {:paths (describe [NonBlankString] "A list of iRODS paths")})
 
 (s/defschema OptionalPaths
   (-> Paths
     (->optional-param :paths)))
 
+(s/defschema DataIds
+  {:ids (describe [UUID] "A list of iRODS data-object UUIDs")})
+
+(s/defschema OptionalPathsOrDataIds
+  (-> (merge DataIds Paths)
+      (->optional-param :paths)
+      (->optional-param :ids)))
```

Comment:
This is the first of several changes which enable the stat-gatherer endpoint to also provide information when given data-object UUIDs, not just paths.